README.md

```diff
@@ -24,24 +24,23 @@ Much of the code is licensed under the Apache 2.0 license. See the [LICENSE](LIC
 We recommend the following hardware to run *stretch-ai*. Other GPUs and other versions of Stretch may support some of the capabilities found in this repository, but our development and testing have focused on the following hardware.
 
 - **[Stretch 3](https://hello-robot.com/stretch-3-product) from [Hello Robot](https://hello-robot.com/)**
-  - When *Checking Software*, `stretch_system_check.py` should report that all software passes.
-  - You can find detailed instructions in the [official guide for updating Stretch 3's software](https://docs.hello-robot.com/0.3/software/updating_software/).
+  - When *Checking Hardware*, `stretch_system_check.py` should report that all hardware passes.
 - **Computer with an NVIDIA GPU**
   - The computer should be running Ubuntu 22.04. Later versions might work, but have not been tested.
   - Most of our testing has used a high-end CPU with an NVIDIA GeForce RTX 4090.
 - **Dedicated WiFi access point**
   - Performance depends on high-bandwidth, low-latency wireless communication between the robot and the GPU computer.
   - The official [Stretch WiFi Access Point](https://hello-robot.com/stretch-access-point) provides a tested example.
-
-To use the learning-from-demonstration (LfD) code you'll need the [Stretch Dexterous Teleop Kit](https://hello-robot.com/stretch-dex-teleop-kit).
+- (Optional) [Stretch Dexterous Teleop Kit](https://hello-robot.com/stretch-dex-teleop-kit).
```

Comment:
Why this change

Reply:
It improves readability. The dex kit is another item on the list.